### PR TITLE
docs: remove back tick before ReversePipe header

### LINKED ISSUE #52594: angular.dev: Remove backtick after ReversedPipe word
### Describe the problem that you experienced

In title of step 1: "Create..."  there is a backtick after ReversePipe word: 

Create the `ReversePipe

### Enter the URL of the topic with the problem

https://angular.dev/tutorials/learn-angular/create-a-pipe#create-the-reversepipe

### Describe what you were looking for in the documentation

_No response_

### Describe the actions that led you to experience the problem

_No response_

### Describe what you want to experience that would fix the problem

Remove backtick after ReversedPipe word or modify md.

### Add a screenshot if that helps illustrate the problem

_No response_

### If this problem caused an exception or error, please paste it here

_No response_

### If the problem is browser-specific, please specify the device, OS, browser, and version

_No response_

### Provide any additional information here in as much as detail as you can

_No response_

### PR DESCRIPTION
**Pull request by sasidharansd**
fixes https://github.com/angular/angular/issues/52594

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #52594

## What is the new behavior?
Backtick is removed before ReversePipe heading

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: adev/src/content/tutorials/learn-angular/steps/24-create-a-pipe/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Alright, it's your turn to give this a try — you'll create the `ReversePipe`:
 
 <docs-workflow>
 
-<docs-step title="Create the `ReversePipe">
+<docs-step title="Create the ReversePipe">
```
Comment (atscott) [inline adev/src/content/tutorials/learn-angular/steps/24-create-a-pipe/README.md:34]:
```suggestion
<docs-step title="Create the `ReversePipe`">
```

I think this should instead add the second tick at the end to match everywhere else in the file.

Comment (sasidharansd) [reply adev/src/content/tutorials/learn-angular/steps/24-create-a-pipe/README.md]: Oops didn't notice that! Updated the PR by adding the missing backtick. Thanks!

**Comment by JeanMeche** [comment]
Hi, can you please squash your commits ? Thx !

**Review by twerske — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 0ffa228165f2d34ed92b01765961998444c3d54b.

**Pull request closed without merging**